### DISCUSSION TITLE
Which version should I use for php5.6?

### DISCUSSION THREAD

**Discussion by bunyamink** [deployphp/deployer#3239] [Help needed]
I want to use deployer package for my web sites but I get error if i use deployer version 7. I think deployer 7 is not suitable for php5.
So, Which deployer version should I use for php5.6?

**Comment by antonmedv** [comment]
I think Deployer v4.

**Reply by bunyamink** [reply]
So, is there documentation for versions prior to version 6? I couldn't find documentation for versions earlier than 6 on deployer.org

**Reply by antonmedv** [reply]
Maybe in git? Don’t know.  It was too long ago.
